wiki/Best_Of/fr.md

```diff
@@ -0,0 +1,322 @@
+[#1]: /wiki/shared/GCrown.png "1st"
+[#2]: /wiki/shared/SCrown.png "2nd"
+[#3]: /wiki/shared/BCrown.png "3rd"
+
+# Best Of série
+
+Sommaire pour toute la série "Best Of".
```

Comment:
Je ne suis pas sûr de comment intégrer `Best Of` dans une phrase en français, je mettrais plutôt `Sommaire de tous les "Best Of".` mais je ne sais pas du tout si c'est correct.